content/billing/managing-billing-for-your-github-account/connecting-an-azure-subscription-to-your-enterprise.md

```diff
@@ -17,7 +17,7 @@ After you connect an Azure subscription, you can also manage your spending limit
 
 ## Connecting your Azure subscription to your enterprise account
 
-To connect your Azure subscription, you must have owner permissions to the subscription.
+At the time of connection, GitHub’s Subscription Permission Validation will request read only access to display the list of available subscriptions. **To select an Azure subscription, you must have owner permissions to the subscription.** If the default tenant does not have the right permissions, you may need to specify a different tenant ID. Learn more about [Microsoft’s authentication flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow#request-an-authorization-code).
```

Comment:
Thanks @cmwilson21! I left a few style updates in the suggestion, but let's also move this line into a note at the end of step 9, which is when it will be most relevant to folks. Feel free to ping me after that's done for another review! ⚡ 

```suggestion
{% data variables.product.company_short %}'s Subscription Permission Validation requests read-only access to display the list of available subscriptions. To select an Azure subscription, you must have owner permissions to the subscription. If the default tenant does not have the right permissions, you may need to specify a different tenant ID. For more information, see [Microsoft identity platform and OAuth 2.0 authorization code flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow#request-an-authorization-code) in Microsoft Docs.
```